readme.md

```diff
@@ -93,6 +114,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 * **Question:** When does `app.use()` get called?
 
 * **Your Answer:**
+Always, it is called every time.  It is just a general request for everything to go through.
```

Comment:
Always called as long as the request has made it to that point in the route chain.